### ISSUE TITLE
Atlas Multi-Z brig break-in issue

### ISSUE THREAD

**Issue by BlueHNT** [BeeStation/NSV13#2641]
## Round ID:
07.05.2024

## Testmerges:
pre gen-pop port

## Reproduction:
The atlas brig jail / armory is easily broken into with an RCD or just making a hole in the ground. It is hilarious, but I would count it as an issue due to it being pretty much unstoppable. This also makes it way too easy for antags to abuse and get a free SMG as they can just climb into armory, ion gun the beepsky, take all the guns they want and leave with both a gun and a security hardsuit.

**Comment by jet-pack-cat** [comment]
Local John Syndicate, Professional larcenist grey-tider here! 

This issue is not exclusive to Atlas, many security departments (and any departments on the deck 2 z-level) are vulnerable. Tycoon, Aetherwhisp, and Gladius are also exposed. Tycoon most notably has its entire security department, hos office, and all, exposed in the darkness of space where nobody looks. It is hilariously unsecured. This makes Serendipity the most secure armory :D! With RCD proof doors, and completely surrounded by Reinforced walls!

Id suggest to have armory and the brigs moved to the top deck, or the above deck be reinforced with difficult to remove tiles. Removing the "move up" verb in gravity also doesn't fix it, as the RCD is a portable anti gravity device, and ladders exist. Though it does fix making escape routes for later, as long as you cant find 10 rods in the brig. You could also remove climbing through lattices, but ladders still exist.

PS. Don't shoot my man Armsky! It can be dragged where you want and doesn't move, or can be bribed with 2 TC.

**Comment by BlueHNT** [comment]
I agree that switching the bar with sec would fix this entire issue. Only minor issue being that hydroponics and kitchen is attached.

Muni will like this change too.

**Comment by SerynEngi** [comment]
Honestly, this is something, along with other issues, that I'd intended to fix later. The problem for the time being that we have 2 mass-mapping change PRs open, one of which being yours, that will cause merging conflicts that I'm just not willing to fight, as this is lower priority.

Instances of people breaking into the armory from above are fairly uncommon on all maps, This is, in part, because while it is easy to break _in_ to the armory, you'd then have to break out, and nothing in the armory will aid you in going through walls.

TL;DR- its on my list of things to at least attempt to fix, but not something I'll deal with soon, as its not a common occurrence in the first place.
